src/ui/component/fileDownloadLink/view.jsx

```diff
@@ -28,6 +30,11 @@ function FileDownloadLink(props: Props) {
     return <span>{label}</span>;
   }
 
+  if (fileInfo && fileInfo.download_path && fileInfo.completed && clicked) {
+    triggerAnalyticsView(uri, 0);
```

Comment:
I'm not sure if we should wait for the download to finish to fire this. If they leave the page before it downloads, it will never fire.

Reply:
Good point. Removed fileInfo.completed.

Reply:
If we aren't tracking that, it can just be fired inside of the `onClick`

Reply:
Yes, Assuming we don't care if unsuccessful downloads are counted as views.

Reply:
We could pass a callback that fires the analytics event here. Then it will happen after the `get` call is successful
https://github.com/lbryio/lbry-desktop/blob/master/src/ui/redux/actions/content.js#L185